### DISCUSSION TITLE
Upscale not working

### DISCUSSION THREAD

**Discussion by therealmaddieli** [lllyasviel/Fooocus#2810] [General]
Tried to upscale this image, both 1.5x and 2x upscale didnot work. prompt "instagram model with pink hair, sitting in a race car"
![Uploading image.png…]() Any idea why?

**Comment by ILikeToasters** [comment]
You would really need to give more info. What happens when you do it? What is showing in the CMD window? Any errors?

**Comment by mashb1t** [comment]
@therealmaddieli for upscaling you need an actual image to upscale, hence the file upload next to it.
You can generate an image, then use this image for upscaling in a second step (manual).

**Comment by therealmaddieli** [comment]
My bad. I got mixed up between upscale and zoom out... now it works fine!
